examples/piggybank/README.md

```diff
@@ -0,0 +1,25 @@
+# Piggy bank web application
+
+The example project included in this repository, serves as a working example of how to integrate with smart contracts on the Concordium blockchain.
+
+## Prerequisites
+
+### Viewing smart contract details
+
+-   [JSON-RPC server](https://github.com/Concordium/concordium-json-rpc/tree/add-get-instance-info) connected to testnet node is accessible at http://127.0.0.1:9095.
+    -   Note that currently the JSON-RPC server must be built from the branch "add-get-instance-info" (link above already targets this branch)
+
+### Submitting transactions
+
+-   Browser wallet extension is installed in google chrome and configured with testnet JSON-RPC.
+    -   Currently, chrome must be running with CORS restrictions disabled.
```

Comment:
I fixed this on the add-get-instance-info branch

Reply:
K, I'll test and remove!